topo_processor/geostore/invoke.py

```diff
@@ -0,0 +1,50 @@
+import json
+from typing import TYPE_CHECKING, Any, Dict
+
+from linz_logger import get_log
+
+if TYPE_CHECKING:
+    from mypy_boto3_lambda import Client
+else:
+    Client = object
+
+
+def invoke_lambda(
+    client: Client, name: str, http_method: str, parameters: Dict[str, str], is_prod: bool = False
+) -> Dict[str, Any]:
+    if not is_prod:
```

Comment:
I dont think we should be hitting any "nonprod" environment that is not ours

Reply:
That's a good point. I've been using their "nonprod" environment to do my tests during development. Now, there is no reason to use this "nonprod" environment. Let's include your option of a `--commit` flag to have a sort of confirmation for any `add`/`delete`

Reply:
It's not their dev environment, it's a clone of prod for us to use for test uploads.

Reply:
Another thing to note is that `-p` or `--prod` works only if you're connected to the AWS Geostore prod account. And non-prod (default mode) works only if you're connected to non-prod. The issue is that the lambda function name are different if you use prod or non-prod. Maybe we could check in the system if the user is authenticated to prod or non prod to identify which function name to use? That way, we get rid of the `-p` or `--prod` flag, and maybe use the `--commit` mode for `add` too?